router/pkg/metric/metric_store.go

```diff
@@ -214,6 +223,11 @@ func (h *Metrics) MeasureInFlight(ctx context.Context, sliceAttr []attribute.Key
 	}
 }
 
+func (h *Metrics) MarkConfigVersionInUse(ctx context.Context, configVersion attribute.KeyValue) {
```

Comment:
Should we add the feature flag name to the metric? This makes it possible to distinguish between them.